README.md

```diff
@@ -37,8 +47,24 @@ resource "onepassword_item" "demo_login" {
 }
 ```
 
-See the [examples](./examples/) directory for a full example.
+## 🛠️ Contributing
+
+For the contribution guidelines, see [CONTRIBUTING.md](/CONTRIBUTING.md).
+
+Still not sure where or how to begin? We're happy to help! You can:
+
+- Join the [Developer Slack workspace](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA), and ask us any questions there.
```

Comment:
As there is only one action to do in this list. Maybe just write it as a sentence? aka
> Still not sure where or how to begin? We're happy to help! Join the [Developer Slack workspace](https://join.slack.com/t/1password-devs/shared_invite/zt-1halo11ps-6o9pEv96xZ3LtX_VE0fJQA), and ask us any questions there.

Reply:
Good point! Fixed in the e57dc32 commit.